ckanext/datastore/tests/test_unit.py

```diff
@@ -43,7 +43,7 @@ def test_pg_version_check(self):
         engine = db._get_engine_from_url(config['sqlalchemy.url'])
         connection = engine.connect()
         assert db._pg_version_is_at_least(connection, '8.0')
-        assert not db._pg_version_is_at_least(connection, '10.0')
+        assert not db._pg_version_is_at_least(connection, '20.0')
```

Comment:
@maxious wouldn't this mean that we support PG 11 to 19? at least in this unit test of course

Reply:
Good for another 10 years or so ;) https://www.postgresql.org/developer/roadmap/
20 seems like a reasonable upper bound for a version that could exist in the future.